packages/itinerary-body/src/stories/OtpUiItineraryBody.story.tsx

```diff
@@ -47,19 +48,24 @@ export const BikeOnlyItinerary = (): ReactElement => (
 );
 
 export const WalkTransitWalkItinerary = (): ReactElement => (
-  <ItineraryBodyDefaultsWrapper itinerary={walkTransitWalkItinerary} />
+  <ItineraryBodyDefaultsWrapper
+    itinerary={walkTransitWalkItinerary}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
```

Comment:
As written, you need to set `trimetMode` to some of these stories for the footer to appear.

Reply:
This being said I think we should get rid of `trimetMode`